src/commoncode/fileutils.py

```diff
@@ -77,7 +77,9 @@ def logger_debug(*args):
         return logger.debug(' '.join(isinstance(a, basestring) and a or repr(a) for a in args))
 
 
-FS_ENCODING = sys.getfilesystemencoding() or sys.getdefaultencoding()
+FS_ENCODING = sys.getfilesystemencoding()
+# normalize the encoding name
+FS_ENCODING = codecs.lookup(FS_ENCODING).name
```

Comment:
Is this still the correct thing to do? I though when using a version of `backports.os` that has https://github.com/pjdelport/backports.os/pull/8 merged, we don't need to do normalization anymore?

Reply:
I see this change is still present with backports.os 0.1.1 in use. Shouldn't your either revert this change, or use the proper fsencode/fsdecode that you mentioned?